rag/advanced_rag/harness/tools/search.py

```
@@ -0,0 +1,289 @@
+        tools.embed_mdl,
+        tools.tenant_ids,
+        target_ids,
+        1,
+        top_n,
+        0.2,
+        vector_similarity_weight=1.0,
+        aggs=False,
+        highlight=False,
+    )
+    if keywords:
+        l = len(kbinfos["chunks"])
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)